doc/source/user_guide/data_types.rst

```diff
@@ -73,6 +73,14 @@ issued::
    float64 to uint8
    array([  0, 128, 255], dtype=uint8)
 
+Warnings can be locally ignored with a context manager::
```

Comment:
Why :: -> : ?
It's part of the rst format no? We use :: everywhere in this file.